app/hotels/src/allHotels/AllHotels.js

```diff
@@ -172,16 +169,23 @@ export default function AllHotelsWithContext(props: Props) {
       }) => (
         <HotelsFilterContext.Consumer>
           {({ filterParams, actions: { setFilter } }) => (
-            <AllHotels
-              {...props}
-              location={location}
-              cityId={cityId}
-              search={searchParams}
-              filter={filterParams}
-              setSearch={setSearch}
-              setFilter={setFilter}
-              setCityIdAndLocation={setCityIdAndLocation}
-            />
+            <GeolocationContext.Consumer>
```

Comment:
I think this (3 nested context, many props) is a great indicator that we are maybe doing too much in this component. Next refactoring should follow... :)

Reply:
Yes, I agree, it has become a little monster 🙈